frontend/awx/administration/execution-environments/hooks/useExecutionEnvironmentsFilters.tsx

```diff
@@ -1,17 +1,24 @@
 import { useDynamicToolbarFilters } from '../../../common/useDynamicFilters';
+import {
+  useCreatedByToolbarFilter,
+  useModifiedByToolbarFilter,
+} from '../../../common/awx-toolbar-filters';
 
 export function useExecutionEnvironmentsFilters() {
+  const createdByToolbarFilter = useCreatedByToolbarFilter();
+  const modifiedByToolbarFilter = useModifiedByToolbarFilter();
   const toolbarFilters = useDynamicToolbarFilters({
     optionsPath: 'execution_environments',
-    preSortedKeys: ['name', 'id'],
+    preSortedKeys: ['name', 'image'],
```

Comment:
We should add `id` back.

Reply:
I would suggest ordering the search dropdown like so : name, id, image, created-by, modified-by